client/jetpack-cloud/components/missing-payment-notification/index.tsx

```diff
@@ -0,0 +1,89 @@
+import { translate } from 'i18n-calypso';
+import { useSelector } from 'react-redux';
+import Notice from 'calypso/components/notice';
+import { getCurrentPartner } from 'calypso/state/partner-portal/partner/selectors';
+import { Invoice } from 'calypso/state/partner-portal/types';
+
+export default function MissingPaymentNotification() {
+	const partner = useSelector( getCurrentPartner );
+
+	if ( ! partner || ! partner.keys ) {
+		return null;
+	}
+
+	// Helper function to check if the date is greater than today
+	const isDateLowerThanToday = ( created: string ) => {
+		const today = new Date();
+		today.setHours( 0, 0, 0, 0 );
+		const createdDate = new Date( Number( created ) * 1000 );
+
+		return createdDate < today;
```

Comment:
Same here, use `moment`. 

```
const today = moment().startOf('day');

return moment.unix( invoice_date ).isBefore( today );
```

**Notes:**

- We no longer use `created` field. We use `effectiveAt` for the invoice date.
- The unix function expects timestamp (seconds), we don't need to rescale it with `*1000`.